desktop/cmp/grid/editors/DateEditor.js

```diff
@@ -17,11 +17,19 @@ export const [DateEditor, dateEditor] = hoistCmp.withFactory({
     memo: false,
     observer: false,
     render(props, ref) {
+        // We need to render the day picker popover inside the grid viewport in order for
+        // `stopEditingWhenCellsLoseFocus` to work properly - otherwise the day picker becomes
+        // unusable due to the grid losing focus and stopping the editing when clicking inside the
+        // picker
+        const portalContainer = props.gridModel.agApi.gridBodyComp.eBodyViewport;
```

Comment:
this seems a bit fragile.  Presumably not part of the public API?  Do we want to make null safe and throw if we can't fine anything there?